flink-core/src/main/java/org/apache/flink/configuration/ExecutionOptions.java

```diff
@@ -55,4 +56,14 @@
 						"throughput")
 				)
 				.build());
+
+	@Documentation.ExcludeFromDocumentation("The option is considered internal and should rather not be used" +
```

Comment:
I would not expose it as an option for now but only set it in the `StreamConfig`.

Reply:
Or in the `StreamNode`?

Reply:
It won't work in `StreamNode`. Only `StreamConfig` is shipped to `StreamTask`.